src/Nethermind/Nethermind.Xdc/Errors/ErrIncomingMessageBlockNotFound.cs

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+
+namespace Nethermind.Xdc.Errors;
+
+public class ErrIncomingMessageBlockNotFound : Exception
```

Comment:
standard naming convention would be: 

```suggestion
public class IncomingMessageBlockNotFoundException : Exception
```

Apply to other exceptions too.